hack/make/binary-daemon

```diff
@@ -17,10 +17,8 @@ copy_binaries() {
 	for file in containerd containerd-shim-runc-v2 ctr runc docker-init rootlesskit rootlesskit-docker-proxy dockerd-rootless.sh dockerd-rootless-setuptool.sh; do
 		cp -f "$(command -v "$file")" "$dir/"
 	done
-
-	# vpnkit is available for x86_64 and aarch64
```

Comment:
Perhaps leave the comment?

Reply:
Not sure that makes sense to keep this comment anymore since it's picked up dynamically based on the target platform. Let me know if that make sense to add a comment in the Dockerfile.

Reply:
My thinking was that it would explain that the binary may not be there (depending on the architecture), which may not immediately be clear (unless you know), so someone may be wondering "why isn't the binary there?"

That said, we could consider moving this one into the loop above (as it looks like it's now doing the same as above?) and put the comment there.

Reply:
Updated with new comment, let me know if that sgty.